pylint/config/option_manager_mixin.py

```diff
@@ -271,6 +271,7 @@ def read_config_file(self, config_file=None, verbose=None):
 
         use_config_file = config_file and os.path.exists(config_file)
         if use_config_file:
+            self.set_current_module(config_file)
```

Comment:
@Pierre-Sassoulas Why is this necessary? Does this add anything other than allowing the test framework to function?

_Ran into this while working on `argparse`._

Reply:
I think it's setting the file that the ``add_message`` will display if we add a message later. I.E. ``[relpath}:1:0: E0013:`` later.